test/run_tests_model_gen_and_load.py

```diff
@@ -134,15 +132,28 @@ def get_tests(
     )
     filtered_tests = []
     for test in tests:
+        vw = vowpalwabbit.Workspace(test.command_line, quiet=True)
+        skip_cmd = False
+        for groups in vw.get_config().values():
+            for group in groups:
+                for opt in group[1]:
+                    if opt.value_supplied and (
+                        opt.experimental
+                        or opt.name == "bfgs"
+                        or (opt.name == "initial_regressor" and opt.value != [])
```

Comment:
where is the --flatbuffer case covered? is flatbuffer tagged as experimental too?

Reply:
Yes flatbuffer is experimental, it actually needs to be checked separately because it segfaults when loading the model

Reply:
we don't need to check flatbuffer files really, they repeat existing tests and the ones in the run tests are for input/output and don't add anything model-wise

Reply:
Yea I meant checked as in removed from the test set